src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationPostService.java

```diff
@@ -119,111 +119,130 @@ public List<Document> getStaticDocs(String auth, String caseType, CaseData caseD
         UploadResponse uploadResponse = null;
         DocumentLanguage documentLanguage = documentLanguageService.docGenerateLang(caseData);
         if (C100_CASE_TYPE.equalsIgnoreCase(caseType)) {
-            List<MultipartFile> files = new ArrayList<>();
-            if (documentLanguage.isGenEng()) {
-                files.add(new InMemoryMultipartFile(
+            uploadResponse = getC100UploadResponse(auth, documentLanguage);
```

Comment:
uploadAndReturnC100StaticDocs (*), try to rename these 2 methods with the mentioned format.